src/payments/index.d.ts

```diff
@@ -28,6 +29,8 @@ export interface Payment {
     redeemVersion?: number;
     scriptTree?: Taptree;
     witness?: Buffer[];
+    bs58EncodeFunc?: BaseConverter['encode'];
+    bs58DecodeFunc?: BaseConverter['decode'];
```

Comment:
This seems like the wrong default. Wouldn't it be `bs58check.decode` / `bs58check.encode`?

The `base-x` library exports a single function that you call with an alphabet, like `BaseConverter('1234567')`, and then this returns the `encode` / `decode` object. So even the way you're accessing `['encode']` and `['decode']` seem off.

Reply:
`BaseConverter` is the object returned by `base-x`'s function.

Reply:
I see. `BaseConverter` is a type, not an object, so this makes sense.